### ISSUE TITLE
Replace chroot with proot on runch.

### ISSUE THREAD

**Issue by lslvr** [Nitrux/tools#1]
[`proot`](https://proot-me.github.io/) is a much more powerful tool for running processes in an isolated environment. It implements `chroot`, `mount --bind`, and `binfmt_misc`.

- It automatically handles the LFHS mount points.
- It can execute binaries of foreign architectures (e.g. `ARMv7 => x86_64`).
- It can bind-mount files or directories.

**Comment by lslvr** [comment]
Even though `proot` seems to be a more powerful tool, it's kind of broken right now. One must do some tricks to get it to work. So a better solution could be creating a wrapper for `chroot` that mimics `proot`.

**Comment by lslvr** [comment]
Some progress has been done here: 6bddf36.

Next steps are:

- Let the user pass arbitrary mount points: `<src>:<mnt>`. Those will be bind-mounted.
- Let the user specify an alternate user id to be used in the `chroot`.

**Comment by lslvr** [comment]
> Let the user pass arbitrary mount points: `<src>:<mnt>`. Those will be bind-mounted.

Implemented on e58f7f1.

**Comment by lslvr** [comment]
> Let the user specify an alternate user id to be used in the `chroot`.

Implemented on 8fcd73d.

---

Flags `-m` and `-u` allow the user to specify mount points and an alternate user to run the processes inside the `chroot` environment. e.g.:

```shell
runch \
    -d /foo/some_dir:/some_dir \
    -u dude \
    my_chroot \
    echo "Hey, all!"
```

**Comment by lslvr** [comment]
`fakechroot` may be a nice option to include.
